pkg/volume/iscsi/iscsi_util.go

```diff
@@ -80,22 +90,27 @@ func (util *ISCSIUtil) MakeGlobalPDName(iscsi iscsiDisk) string {
 }
 
 func (util *ISCSIUtil) AttachDisk(b iscsiDiskBuilder) error {
-	devicePath := strings.Join([]string{"/dev/disk/by-path/ip", b.portal, "iscsi", b.iqn, "lun", b.lun}, "-")
-	exist := waitForPathToExist(devicePath, 1)
+	var devicePath string
+	if b.iface == "default" {
+		devicePath = strings.Join([]string{"/dev/disk/by-path/ip", b.portal, "iscsi", b.iqn, "lun", b.lun}, "-")
+	} else {
+		devicePath = strings.Join([]string{"/dev/disk/by-path/pci", "*", "ip", b.portal, "iscsi", b.iqn, "lun", b.lun}, "-")
```

Comment:
I don't have the setup. Any reference in [udev](https://github.com/systemd/systemd/blob/master/src/udev/udev-builtin-path_id.c) I can see this device path?